### PR TITLE
[IMP] l10n_be: Added translations file nl.po and l10n_be.pot for new …

### PR DESCRIPTION
**Pull request by mja-odoo**
…model account.tax.report.line

- translation provided in Dutch

task : 2042430
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_be/i18n_extra/l10n_be.pot [hunk 1]
```diff
@@ -1,1664 +1,1847 @@
...
+"PO-Revision-Date: 2019-08-01 07:15+0000\n"
+"Last-Translator: \n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: \n"
 
 #. module: l10n_be
+#: model:account.tax,name:l10n_be.1_attn_VAT-IN-V83-00
```
Comment (mart-e) [inline addons/l10n_be/i18n_extra/l10n_be.pot]: don't add this, this is the point of the l10n_multilang module, you translate the templates, not the generated records

File: addons/l10n_be/i18n_extra/nl.po [hunk 2]
```diff
@@ -4,1665 +4,1849 @@
...
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Generator: Poedit 1.8.12\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
 
 #. module: l10n_be
+#: model:account.tax,name:l10n_be.1_attn_VAT-IN-V83-00
 #: model:account.tax.template,name:l10n_be.attn_VAT-IN-V83-00
 msgid "0% Biens d'investissement"
-msgstr "0% Investeringsgoederen"
```
Comment (mart-e) [inline addons/l10n_be/i18n_extra/nl.po]: why are you removing translations?

File: odoo/tools/translate.py [hunk 3]
```diff
@@ -603,7 +603,7 @@ def __iter__(self):
             translation = entry.msgstr
             found_code_occurrence = False
             for occurrence, line_number in entry.occurrences:
-                match = re.match(r'(model|model_terms):([\w.]+),([\w]+):(\w+)\.(\w+)', occurrence)
```
Comment (jco-odoo) [inline odoo/tools/translate.py]: @jva-odoo @mja-odoo Why/Where do we have xml_ids with -?

Comment (jva-odoo) [reply odoo/tools/translate.py]: @jco-odoo https://github.com/odoo/odoo/blob/master/addons/l10n_be/data/account_tax_template_data.xml#L4

Comment (mart-e) [reply odoo/tools/translate.py]: Done at 6d98f838571964835e77208cacfc628786cb01

Comment (jva-odoo) [reply odoo/tools/translate.py]: @mart-e Ok 👍 
This is a temporary fix in this brach so now we remove it.
Thank you

**Comment by mart-e** [comment]
Why do you target the saas-12.5? It is no longer an active branch.

**Comment by jco-odoo** [comment]
@oco-odoo

**Comment by oco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**